docs/book/src/tasks/kubeadm-control-plane.md

```diff
@@ -0,0 +1,28 @@
+# Kubeadm control plane
+
+Using the Kubeadm control plane type to manage a control plane provides several ways to upgrade control plane machines.
+
+## Upgrading
+
+The high level steps to fully upgrading a workload cluster are to first upgrade the control plane and then upgrade
+the worker machines.
+
+### Upgrading workload machines managed by a `MachineDeployment`
+
+It is recommended to manage workload machines with a `MachineDeployment` as this will transparently manage `MachineSet`s
+and `Machine`s to allow for a seamless scaling experience. A modification to the `MachineDeployment`s spec will begin a
+rolling update of the workload machines.
+
+### How to upgrade the underlying machine image
+
+To upgrade the control plane machines underlying machine images, the `MachineTemplate` resource referenced by the
+`KubeadmControlPlane` must be changed. Since `MachineTemplate` resources are immutable, the recommended approach is to
+copy the existing `MachineTemplate`, modify the necessary values, create the new `MachineTemplate` on the management
```

Comment:
"modify the necessary values" might confuse the reader, is it possible to link somehow this sentence with the next paragraph (where there is the explanation of what should be modified)
probably a simple "(see next paragraph for an explanation of the possible changes)" might suffice...